res/values-zh-rCN/strings.xml

```diff
@@ -1029,6 +1029,8 @@
     <string name="adaptive_sleep_camera_lock_summary" msgid="8417541183603618098">"必须解锁相机才能使用屏幕感知功能"</string>
     <string name="auto_rotate_summary_no_permission" msgid="1025061139746254554">"人脸检测功能需要获得相机使用权限。点按即可管理“设备个性化服务”的权限"</string>
     <string name="auto_rotate_manage_permission_button" msgid="2591146085906382385">"管理权限"</string>
+    <string name="touch_sensitivity_title">触控增强</string>
+    <string name="touch_sensitivity_summary">增强贴膜时的触控体验</string>
```

Comment:
I can't promise these won't get removed when I merge a patch

Reply:
It'd be better to add these in yaap_strings even tho they are an AOSP string

Reply:
resolved